### ISSUE TITLE
[Bug] AnkiDroid removes custom scheduling when syncing

### ISSUE THREAD

**Issue by matejdro** [ankidroid/Anki-Android#13009]
###### Reproduction Steps

1. Add something to the "Custom scheduling" field in the Anki Desktop 2.1.55
2. Sync desktop
3. Sync mobile
4. Sync desktop again

###### Expected Result

Custom scheduling field should remain filled

###### Actual Result

Field is cleared.

Now, I understand that that AnkiDroid does not use that field yet and it will take some work to support it (#12620). However, this ticket is only about sync. AnkiDroid should not be deleting that data, even if it is not making use of it.

###### Debug info

```
AnkiDroid Version = 2.16alpha29

Android Version = 13

Manufacturer = Google

Model = Pixel 7 Pro

Hardware = cheetah

Webview User Agent = Mozilla/5.0 (Linux; Android 13; Pixel 7 Pro Build/TQ1A.221205.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/108.0.5359.128 Mobile Safari/537.36

ACRA UUID = 85314e7b-0d43-4b96-a861-e60e9dd7f848

Scheduler = std2

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

###### Research
*Enter an [x] character to confirm the points below:*

- [X] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [X] I have searched for similar existing issues here and on the user forum
- [x] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by mikehardy** [comment]
Hey there - I just merged #12897 and will do a new alpha with it shortly, but I notice you are not using the new backend + v3 scheduler. You do not appear to be on remotely modern alpha version either - 29 is quite old

The v2 scheduler shouldn't delete things, but it *may* be that for folks wanting to use the newest features of v3 scheduler we have to siply document that it requires v3 scheduler here.

Can you re-try your reproduction after updating to the latest release, and if it still reproduces, go into settings -> advanced and turn on support for the new backend + v3 scheduler? https://github.com/ankidroid/Anki-Android/releases

You will be in all new codepaths at that point and this may just work at that point

**Comment by BrayanDSO** [comment]
Duplicate of #12866

**Comment by matejdro** [comment]
Huh that's weird, Play store shows no updates (and I'm in the beta branch). Will install latest alpha manually, thanks.

**Comment by mikehardy** [comment]
@matejdro that is true - Google Play Store for even the beta channel is denying (and rightly so) any updates from us at all until we fix + release the storage migration for #5304 - it is in progress but until then prebuild apks of our latest code areonly available here on our releases page
